newpr.py

```diff
@@ -160,22 +167,16 @@ def remove_label(self, label):
         try:
             result = self.api_req("DELETE", self.remove_label_url % (self.owner, self.repo, self.issue, label), {})
         except urllib2.HTTPError, e:
-            #if e.code == 201:
-            #    pass
-            #else:
-            #    raise e
-            pass
+            if e.code == 201:
+               pass
+            else:
+               raise e
 
     def get_labels(self):
         if self._labels is not None:
             return self._labels
-        try:
-            result = self.api_req("GET", self.get_label_url % (self.owner, self.repo, self.issue))
-        except urllib2.HTTPError, e:
-            if e.code == 201:
-                pass
-            else:
-                raise e
```

Comment:
I removed this because I don't think we'll ever get a 201 code for a GET request.